darts/tests/models/forecasting/test_ensemble_models.py

```diff
@@ -183,8 +189,35 @@ def test_call_predict_global_models_multivariate_input_with_covariates(self):
 
     @unittest.skipUnless(TORCH_AVAILABLE, "requires torch")
     def test_input_models_mixed(self):
+        # NaiveDrift is local, RNNModel is global
+        naive_ensemble = NaiveEnsembleModel([NaiveDrift(), RNNModel(12, n_epochs=1)])
+        # ensemble is neither local, nor global
+        self.assertFalse(naive_ensemble.is_local_ensemble)
+        self.assertFalse(naive_ensemble.is_global_ensemble)
+
+        # ensemble contains one local model, no support for multiple ts fit
+        with self.assertRaises(ValueError):
+            naive_ensemble.fit([self.series1, self.series2])
+
+    @unittest.skipUnless(TORCH_AVAILABLE, "requires torch")
+    def test_mixed_models_with_covariates(self):
```

Comment:
Using regression models with different covariates raise an error. 
e.g. take one regression model with lags_past_covariates, and another one with lags_future_covariates.
Now pass both past_covariates and future_covariates to the ensemble model's fit/predict. 

```
from darts.datasets import AirPassengersDataset
from darts.models import NaiveEnsembleModel
from darts.models import LinearRegressionModel


m_pc = LinearRegressionModel(lags=1, lags_past_covariates=[-1])
m_fc = LinearRegressionModel(lags=1, lags_future_covariates=[-1])
model = NaiveEnsembleModel(models=[m_pc, m_fc])
series = AirPassengersDataset().load()
model.fit(series=series, past_covariates=series, future_covariates=series)
model.predict(n=1, series=series, past_covariates=series, future_covariates=series)
```
Maybe we need to update RegressionModel's properties `supports_past/future_covariates` to be based on what lags the user gave at model creation?